### ISSUE TITLE
Windows (~touch screen) file selection 

### ISSUE THREAD

**Issue by aswarren** [PATRIC3/patric3_website#2088]
Many users at Midwestern (6-8) experienced a problem selecting a single read file in Chrome. They would make the selection in the widget, and it wouldn't show up in the field. All seemed to be windows users, many seemed to be touchscreen users as well.

1. 
![image](https://user-images.githubusercontent.com/229173/44235157-8f77b500-a176-11e8-81c9-87e1ee3c9c3d.png)

2. 
![image](https://user-images.githubusercontent.com/229173/44235180-a5857580-a176-11e8-86bf-050b73fc52ff.png)

3. 
![image](https://user-images.githubusercontent.com/229173/44235202-b8984580-a176-11e8-9d22-46d346356b18.png)

**Comment by aswarren** [comment]
This might take someone capable of debugging dojo widgets. Getting such a person on a Windows box seems like a challenge.

**Comment by olsonanl** [comment]
We need to be including windows testing ... no reason not to with VirtualBox and freely available VMs from Microsoft for all their browsers.

**Comment by aswarren** [comment]
I was able to make this work by selecting the file once, seeing it wasn't there, going back in, going up to the directory above it, going back down, and selecting the file again.

**Comment by dawenx** [comment]
@chris-c-thomas do you have Windows w/ touch screen?

**Comment by olsonanl** [comment]
Doesn’t the chrome development support allow emulation of touchscreen?

On Feb 4, 2019, at 1:52 PM, Dawen Xie <notifications@github.com<mailto:notifications@github.com>> wrote:


@chris-c-thomas<https://github.com/chris-c-thomas> do you have Windows w/ touch screen?

**Comment by dmachi** [comment]
Chrome does support this for testing to the extent one is testing with Chrome itself.  See https://developers.google.com/web/tools/chrome-devtools/device-mode/

For testing other browsers on windows, it seems support varies.  https://social.msdn.microsoft.com/Forums/ie/en-US/03b6cd67-f0d2-4fb6-8d07-3ac799993383/how-do-i-emulate-touch-events-without-using-touch-screen-?forum=wpf

Several recommendations on that page and others I searched recommend using http://multitouchvista.codeplex.com/ <http://multitouchvista.codeplex.com/> .

**Comment by chris-c-thomas** [comment]
@dawenx - I do not. Only desktop and VM on MacOS but @olsonanl is right, it does to an extent.

https://developers.google.com/web/tools/chrome-devtools/device-mode/

**Comment by chris-c-thomas** [comment]
@dmachi hah your response showed up right as I clicked comment.

**Comment by dawenx** [comment]
Yes, there is. If there's a touch screen, it would be the best option compared to simulation which is an approximation. anyway, i was going through un-assigned issues and wanted to know what kind of dev. environment people have.
